sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -162,20 +215,24 @@ mod tests {
                 })
             })
             .collect();
-        unsorted_recent_blockhashes.shuffle(&mut thread_rng());
+        blocks.shuffle(&mut thread_rng());
 
+        let mut unsorted_recent_blockhashes: Vec<_> = blocks
```

Comment:
nits: I think we can move this code [below](https://github.com/solana-labs/solana/pull/8609/files#diff-5b0e068e20dee2cc44d0abaf604eaa1cR230) as it's no longer needed to reduce context.